vignettes/join-keys.Rmd

```diff
@@ -212,22 +227,19 @@ join_keys(td_fk) <- join_keys(
   join_key("ds2", keys = c("V", "W")),
   join_key("ds3", keys = c("V")),
   # Foreign keys
-  join_key("ds2", "ds1", c("W" = "X")),
-  join_key("ds3", "ds2", c("V" = "V"))
+  join_key("ds1", "ds2", c("X" = "W")),
+  join_key("ds2", "ds3", c("V" = "V"))
 )
 
-# The parent of ds2 and ds3 is ds1
-parents(td_fk) <- list(ds2 = "ds1", ds3 = "ds2")
-
 join_keys(td_fk)
 ```
 
 ### Implicit Relationships
 
-Two datasets that share common _foreign_ keys to the same _parent_ dataset have an implicit relationship between them that is modeled and accessible in `join_keys`.
+Two datasets that share common _foreign_ keys to the same _parent_ dataset have an implicit relationship between them that is modeled and accessible in _joining keys_.
 
-This is a special inferred relationship from existing `join_keys` that does not need to be explicitly defined and can be seamlessly accessible just as any other foreign key.
-As any other foreign key it can be overwritten.
+This is a special inferred relationship from existing foreign keys that does not need to be explicitly defined and can be seamlessly accessible just as any other foreign key.
+As any other foreign key they can be overwritten.
```

Comment:
```suggestion
This is a special relationship that is inferred from existing foreign keys.
It does not need to be explicitly defined but it can be accessed and overwritten just as any other foreign key.
```